tls/src/keys.rs

```diff
@@ -397,6 +398,22 @@ pub enum AttestConfig {
     },
 }
 
+/// The result of measurement appraisal
+#[derive(Clone, Debug, Deserialize)]
+pub enum SprocketsResult {
```

Comment:
I think a better name would be `MeasurementStatus`, and then the variants could be `Verified` and `Unverified`.

I find it confusing when a type includes `Result` in the name even though it is not a `Result`.